storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5688,6 +5688,30 @@ static int rocksdb_init_internal(void *const p) {
   }
 
   if (rocksdb_persistent_cache_size_mb > 0) {
+    // TODO: This is the limitation in RocksDB.
+    // There is an assertion for incorrect options.
+    //
+    // utilities/persistent_cache/block_cache_tier.cc:33
+    // status = opt_.ValidateSettings();
+    // assert(status.ok()); // <- THIS IS
+    // if (!status.ok()) {
+    // ...
+    // }
+    //
+    // BEGIN
+    if (rocksdb_persistent_cache_size_mb < 100) {
+      LogPluginErrMsg(ERROR_LEVEL, 0, "Invalid persistent cache options");
```

Comment:
Would be nice to write something like
`Invalid value for rocksdb_persistent_cache_size_mb. It has  to be at least XYZ`

also please do not hardcode `100` in such a case twice, rather use `static constexpr int persistent_cache_size_mb_min = 100;` and then use it.